task/04-arrays-tasks.js

```diff
@@ -454,7 +529,15 @@ function getIdentityMatrix(n) {
  *     3, 3   => [ 3 ]
  */
 function getIntervalArray(start, end) {
-   throw new Error('Not implemented');
+   let initionValue = start
+   const arr = []
+
+   while (initionValue <= end) {
+      arr.push(initionValue)
+      initionValue++
+   }
+
+   return arr
```

Comment:
Can you come up with solution without using `while` loop there?

Hint: look at `Array.from` method